sql/catalyst/src/test/scala/org/apache/spark/sql/connector/catalog/InMemoryTable.scala

```diff
@@ -161,7 +162,11 @@ class InMemoryTable(
           case (v, t) =>
             throw new IllegalArgumentException(s"Match: unsupported argument(s) type - ($v, $t)")
         }
-      case BucketTransform(numBuckets, ref, _) =>
+      case BucketTransform(numBuckets, ref) =>
```

Comment:
I think we can have a single `BucketTransform.unapply`, to match both `BucketTransform` and `SortedBucketTransform`, so that we can have a single case here and avoid duplicated code.